src/EditorFeatures/Core/IntelliSense/AsyncCompletion/CommitManager.cs

```diff
@@ -237,6 +241,16 @@ public IEnumerable<char> PotentialCommitCharacters
                 return new AsyncCompletionData.CommitResult(isHandled: true, AsyncCompletionData.CommitBehavior.None);
             }
 
+            // Specifically for snippets, we use reflection to try and invoke the LanguageServerSnippetExpander's
+            // TryExpand method and determine if it succeeded or not.
+            if (SnippetCompletionItem.IsSnippet(roslynItem))
+            {
+                var lspSnippetText = change.Properties[SnippetCompletionItem.LSPSnippetKey];
+
+                _roslynLSPSnippetExpander.TryExpand(change.TextChange.Span, lspSnippetText, _textView, triggerSnapshot);
```

Comment:
nit: if return value is not checked. consider just making "Expand" and not "TryExpand"